src/Microsoft.DocAsCode.Build.Engine/TemplateProcessor.cs

```diff
@@ -215,8 +214,10 @@ private static void SaveManifest(List<ManifestItem> manifest, string outputDirec
 
             var toc = context.GetTocInfo();
             var manifestObject = GenerateManifest(context, manifest);
+            // TODO: Not serialize manifest here
```

Comment:
I also move it ... Do we have conflict?
[https://github.com/dotnet/docfx/pull/477/files#diff-f0ef6832e40154ac8de26751cb8e19c3R83](https://github.com/dotnet/docfx/pull/477/files#diff-f0ef6832e40154ac8de26751cb8e19c3R83)